### ISSUE TITLE
Package name normalisation in pkglts

### ISSUE THREAD

**Issue by SamuelAndresPascal** [revesansparole/pkglts#240]
I suppose [this dev](https://github.com/revesansparole/pkglts/commit/ad9155a9164bbe13265c891dcd8c71d4a39d3c81) was based on https://packaging.python.org/en/latest/specifications/name-normalization/ .

In pkglts, this developments leads to write only normalized package names in pypi and conda dependency management files.

Nevertheless, the specification states :

1. The underscore is quite legal to appear in package names.

So, in other terms, pkglts forbids to use legal packages whose name contains the underscore character. But is it really the pkglts purpose to restrains the user in such a way ? This behavior change is *breaking* and leads to override and manually maintain the pypi and conda dependency configurations to keep using legal packages whose names contain the underscore character.

2. The normalization process "*should be done before lookups and comparisons*."

I think those cases are not in the pkglts perimeter which only factorizes and dispatches the package names supplied by the user into the pypi and conda templated dependency files. It is not up to pkglts to make operations *before lookup and comparisons*, but only to avoid to directly write pypi and/or conda dependency configurations. pkglts should allow all the legal package names (including not normalized ones) as long as pypi and conda allow them. 

The day a package name will not be considered *legal* by pipy and/or conda, it is up to those tools to manage the issue, and up to the user to make the appropriate changes in the `pkg_cfg.json` requirement section.

------

In conclusion, I think we could remove the package name normalization in pipy/conda dependency management files with the best expected benefit.
